lib/collapsible.php

```
@@ -0,0 +1,68 @@
+
+add_filter('mce_buttons', 'zw_collapsible_editor_button', 10, 2);
+
+/**
+ * Rewrites sections into their canonical form before they are stored. This runs
+ * before the theme's kses allowlist, which would otherwise drop a stray h1
+ * together with the title class.
+ */
+function zw_collapsible_sanitize_post_data(array $data): array
+{
+    if (isset($data['post_content']) && str_contains($data['post_content'], 'collapsible')) {
+        $data['post_content'] = wp_slash(CollapsibleNormalizer::normalize(wp_unslash($data['post_content'])));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Streekomroep\CollapsibleNormalizer' in method 'zw_collapsible_sanitize_post_data'. (undefined)

(StaticAccess)